webapp-django/crashstats/crashstats/models.py

```diff
@@ -268,21 +260,12 @@ def config_from_configman():
         default=socorro.external.boto.crash_data.TelemetryCrashData,
     )
 
-    config = configuration(
+    return configuration(
         definition_source=definition_source,
         values_source_list=[
             settings.SOCORRO_IMPLEMENTATIONS_CONFIG,
         ]
     )
-    # The ReprocessingOneRabbitMQCrashStore crash storage, needs to have
-    # a "logger" in the config object. To avoid having to use the
-    # logger set up by configman as an aggregate, we just use the
-    # same logger as we have here in the webapp.
-    config.queuing.logger = logger
-    config.priority.logger = logger
-    config.crashdata.logger = logger
-    config.telemetrydata.logger = logger
-    return config
```

Comment:
A while back, we moved the logger and metrics out of configuration. So we don't need this anymore.